### PR TITLE
QC For Deciding When to Take Pensions

### PR DESCRIPTION
**Pull request by CurtisUnderwood**
### Description

List of proposed changes:

- Swapped JPGs out with PNGs to update graph typo
- Fixed content typos across EN and FR

### Additional Notes

### CODE REVIEW

File: frontend/public/locales/fr/learn/deciding-when-to-start-your-public-pensions.json [hunk 1]
```diff
@@ -48,12 +48,12 @@
...
-    "p3": "Lisez <a2>Fred décide quand prendre ses pensions publiques</a2> pour savoir ce qui guide ses choix.",
-    "smart-tip": "<strong>Conseil judicieux\u00a0:</strong> le Canadien moyen qui prend sa retraite à 60 ans pourrait perdre   100 000 dollars de revenu de retraite. Chaque mois où vous reportez le versement de votre pension, vous augmentez votre versement mensuel. Si vous reportez votre pension de retraite du RPC de 60 à 70 ans, vous pourriez plus que doubler votre montant mensuel. Votre pension sera modifiée pour tenir compte de l'inflation future."
+    "p3": "Lisez <a2>Fred décide quand commencer à recevoir ses pensions publiques</a2> pour savoir ce qui guide ses choix.",
+    "smart-tip": "<strong>Conseil judicieux\u00a0:</strong> Le Canadien moyen qui prend sa retraite à 60 ans, pourrait perdre   100 000 $ de revenu de retraite. Chaque mois où vous reportez le versement de votre pension, vous augmentez votre versement mensuel. Si vous reportez votre pension de retraite du RPC de 60 à 70 ans, vous pourriez plus que doubler votre montant mensuel. Votre pension sera modifiée pour tenir compte de l'inflation future."
```
Comment (sebastien-comeau) [inline frontend/public/locales/fr/learn/deciding-when-to-start-your-public-pensions.json:56]: `   100 000 $` should be ` 100\u0a00000\u0a00$`

File: frontend/public/locales/fr/learn/deciding-when-to-start-your-public-pensions.json [hunk 2]
```diff
@@ -126,9 +126,9 @@
...
+      "description": "Le tableau indique les montants mensuels de la pension de la SV à l'âge de 75 ans, en fonction de l'âge auquel vous commencez à recevoir votre pension. Il montre que plus vous attendez pour commencer à recevoir votre pension, plus vous recevrez d'argent. Vous pouvez commencer à recevoir votre pension de la SV à 65 ans pour le montant le plus faible ou à 70 ans pour le montant le plus élevé. Si vous choisissez de retarder le versement de votre pension et de recevoir une pension plus élevée, l'augmentation automatique de 10 % à l'âge de 75 ans vous permettra d'obtenir plus d'argent.",
```
Comment (sebastien-comeau) [inline frontend/public/locales/fr/learn/deciding-when-to-start-your-public-pensions.json:129]: `10 %` should be `10\u00a0%`

File: frontend/public/locales/fr/learn/deciding-when-to-start-your-public-pensions.json [hunk 3]
```diff
@@ -48,12 +48,12 @@
...
+    "p1": "Quel que soit le nombre de fois où vous avez changé d'emploi, en travaillant au Canada, vous avez contribuer au Régime de pensions du Canada (RPC) ou au <a1>Régime de rentes du Québec (RRQ).</a1> Le montant de votre pension de retraite du RPC dépend du montant et de la durée de vos contributions. Le RPC vous cotisations de choisir la date de début de votre pension de retraite, et chaque mois que vous reportez augmente votre montant mensuel. Le montant mensuel le plus élevé que vous puissiez recevoir est atteint à l'âge de 70 ans, après quoi il n'y a plus d'avantage à attendre. Si vous avez besoin d'argent plus tôt, vous pouvez commencer à recevoir votre pension dès l'âge de 60 ans, mais avec une réduction permanente.",
```
Comment (gregory-j-baker) [inline frontend/public/locales/fr/learn/deciding-when-to-start-your-public-pensions.json:53]: To remain consistent with other links in the application, the period should be moved outside of the `<a1>` here..

ex:

```
<a1>Régime de rentes du Québec (RRQ)</a1>.
```

**Review by gregory-j-baker — CHANGES_REQUESTED** [review]

File: frontend/public/locales/fr/learn/deciding-when-to-start-your-public-pensions.json (hunk 3 above)
Comment (CurtisUnderwood) [reply frontend/public/locales/fr/learn/deciding-when-to-start-your-public-pensions.json]: Fixed

**Review by sebastien-comeau — APPROVED** [review]

**Review by gregory-j-baker — APPROVED** [review]

**Pull request merged**